Source/MonoGame.Extended/BitmapFonts/BitmapFont.cs

```diff
@@ -44,28 +44,48 @@ internal static IEnumerable<int> GetUnicodeCodePoints(string s)
 
         public Size GetSize(string text)
         {
+            var totalWidth = 0;
             var width = 0;
+            var totalHeight = 0;
             var height = 0;
+
+            const int newlineCodePoint = 10;
```

Comment:
Is there any reason why this line couldn't be `const int newlineCodePoint = '\n';`?